charts/enterprise-kyverno-operator/templates/pre-delete-hook.yaml

```diff
@@ -49,7 +49,7 @@ spec:
               kubectl -n {{ include "enterprise-kyverno.namespace" . }} delete kyvernoadapters cis-adapter-config;
               {{- end }}
               {{- if .Values.kyverno.createCR }}
-              kubectl -n {{ include "enterprise-kyverno.namespace" . }} delete kyvernoes kyverno;
+              kubectl -n {{ include "enterprise-kyverno.namespace" . }} delete kyvernoconfigs kyverno;
```

Comment:
Here as well `include "enterprise-kyverno.namespace"`, shall we rename it to be consistent?

Reply:
Did not change code (internal variables).